t/buy-sig-libp2p-local-integration-test.ss

```diff
@@ -0,0 +1,219 @@
+(export #t)
```

Comment:
I'm kindof wary of continuing to copy & paste the bulk of the logic for these integration tests; it might be prudent to try to factor some of this out into a function.

Reply:
Refactored the test setup boilerplate. The other areas seem to be specific to the integration test itself (options we pass in etc...)